npm/pkg/transport/pb/transport.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package pb;
+option go_package = "github.com/Azure/azure-container-networking/npm/pkg/transport/pb;pb";
```

Comment:
For package names, it might be better to communicate domain. In this case `pb` seems a bit generic and it might clash with other packages. Thoughts `dataplane` or `networkpolicy`? Seems like that is the domain of the services and types below.

Reply:
Since it is local to the transport package I don't think it should matter

Reply:
@ramiro-gamarra Thoughts?

Reply:
Not sure I understand the part about local. The imported package would still be `pb` and referenced as such across the code. If the client had a second grpc service to interact with for example, and its package was under `pkg/foo/pb`, it would also be imported as `pb` but would conflict with the other one, even if they are unrelated paths. Maybe I'm missing a detail, let me know

Reply:
I see your point now. @vakalapa thoughts? Since this proto is common across both cp and dp how would we name it
Maybe we restructure as we get it it? @vakalapa what are your thoughts on this?

Reply:
Ack, i can see that confusion arising, and we will be expanding later on, so having a specific pkg name instead of pb would help for sure, 

Other option is to consolidate all protos into a central location under like below (calico does this) so all other pks can access these types without any cyclic issues 

>npm/pkg/protos

Reply:
Let me see what I can do

Reply:
Done